Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM ubuntu:xenial
+
+# Install programs for runtime use (e.g. by Bazel, Node-GYP)
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+      apt-transport-https \
+      bash \
+      ca-certificates \
+      curl \
+      g++ \
+      git \
+      lbzip2 \
+      make \
+      nodejs \
+      openjdk-8-jdk \
+      python \
+      unzip \
+      xz-utils
+
+# Install Chrome for Selenium
+#  Reference: https://tecadmin.net/setup-selenium-chromedriver-on-ubuntu/
+RUN curl -sS -o - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add
+RUN echo "deb [arch=amd64]  http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
+RUN apt-get -y update && apt-get -y install google-chrome-stable
+# Install Selenium
+RUN curl https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.zip -o  /usr/bin/chromedriver_linux64.zip
+RUN unzip /usr/bin/chromedriver_linux64.zip -d /usr/bin/chromedriver
+RUN chmod +x /usr/bin/chromedriver
+
+# Set up workspace
+ENV WORKSPACE /usr/src/app
+RUN mkdir -p $WORKSPACE
+WORKDIR $WORKSPACE
+
+# Install ktlint
+RUN (cd /usr/bin/ && curl -L -s -O https://github.com/pinterest/ktlint/releases/download/0.35.0/ktlint && cd -)
+RUN chmod +x /usr/bin/ktlint
```

Comment:
Nit: this could be done in the same RUN command using another `&&`

Reply:
Done.